tests/rust_usage_test/tests/flexbuffers_tests/rwyw.rs

```diff
@@ -475,7 +475,7 @@ fn deserialize_newtype_str() {
     #[derive(Deserialize)]
     struct Foo<'a>(&'a str);
     let data = [5, b'h', b'e', b'l', b'l', b'o', b'\0', 6, 5 << 2, 1];
-    let r = Reader::get_root(&data).unwrap();
+    let r = Reader::get_root(data.as_ref()).unwrap();
```

Comment:
is going from `&data` to `data.as_ref()` necessary? I think this should be reverted because this PR shouldn't break this code and this change is otherwise irrelevant

Reply:
This change has to be made b/c `&data` is of type `[u8; 10]`, instead of a `[u8]` type. 

An `InternalBuffer` cannot be implemented for `[u8; 10]` since it has the `slice` method.